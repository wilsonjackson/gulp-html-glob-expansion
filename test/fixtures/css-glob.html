<html>
<head>
	<title>You Always Look Fine</title>

	<link type="text/css" rel="stylesheet" href="/assets/*.css">
</head>
</html>
