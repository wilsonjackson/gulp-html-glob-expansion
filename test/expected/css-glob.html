<html>
<head>
	<title>You Always Look Fine</title>

	<link type="text/css" rel="stylesheet" href="/assets/file.css">
	<link type="text/css" rel="stylesheet" href="/assets/otherfile.css">
</head>
</html>
